src/tools/interop/interop.cpp

```diff
@@ -24,7 +24,7 @@ const QUIC_API_TABLE* MsQuic;
 HQUIC Registration;
 int EndpointIndex = -1;
 uint32_t TestCases = QuicTestFeatureAll;
-uint32_t WaitTimeoutMs = 5000;
+uint32_t WaitTimeoutMs = 10000;
```

Comment:
This really necessary?

Reply:
I was wondering if part of the reason the high RTT/loss tests fail is that the file doesn't finish transferring within 5 seconds. Some of those files are over 25 MB.

Reply:
Ok.